portal/v2/src/SSHModal.tsx

```diff
@@ -0,0 +1,239 @@
+import { useId, useBoolean } from "@fluentui/react-hooks"
+import {
+  Modal,
+  getTheme,
+  mergeStyleSets,
+  FontWeights,
+  IIconProps,
+  Dropdown,
+  IDropdownOption,
+  TextField,
+  MessageBar,
+  MessageBarType,
+  Stack,
+  FontSizes,
+} from "@fluentui/react"
+import { PrimaryButton, IconButton } from "@fluentui/react/lib/Button"
+import axios, { AxiosResponse } from "axios"
+import React, {
+  useState,
+  useImperativeHandle,
+  useEffect,
+  forwardRef,
+  MutableRefObject,
+} from "react"
+
+const cancelIcon: IIconProps = { iconName: "Cancel" }
+const copyIcon: IIconProps = { iconName: "Copy" }
+
+const machineOptions = [
+  { key: 1, text: "master-1" },
```

Comment:
can these mirror how they are named on the cluster? `master-0,1,2`?

Reply:
Fixed